lerobot/scripts/eval.py

```diff
@@ -210,8 +210,8 @@ def eval_policy(
     videos_dir: Path | None = None,
     return_episode_data: bool = False,
     start_seed: int | None = None,
-    enable_progbar: bool = False,
-    enable_inner_progbar: bool = False,
+    enable_progbar: bool = True,
```

Comment:
Can I check reasoning for changing these defaults? I believe you wanted to see the progbar during training, but in that case, why not pass the kwarg you want from `train.py`?

Reply:
Yes I want to see the progbar during training. OK I will add them to `default.yaml` and pass the kwarg from `train.py`, that's reasonable